docs/spec_draft.md

```diff
@@ -709,6 +710,58 @@ lexicographic comparison on the (real, imaginary) pairs with corner cases TBD.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.multiply
+
+### Semantics
+
+Performs element-wise product of two tensors `lhs` and `rhs` and produces a
+`result` tensor. For integer element types, if the element-wise product has an
+unsigned/signed overflow/underflow, the result is implementation-defined and one
+of the following:
+
+  * mathematical result modulo $2^n$, where n is the bit width of the result,
+  for unsigned overflow/underflow. For signed integer overflow/underflow, wraps
+  the result around the representable range $[-2^{n-1},\ \ 2^{n-1} - 1]$.
+  * saturation to $2^{n-1} - 1$ (or $-2^{n-1}$) for signed overflow (or signed
+  underflow) and saturation to $2^n - 1$ (or $0$) for unsigned overflow (or
+  unsigned underflow).
+
+For floating-point element types, it implements the `multiplication` operation
+from the IEEE-754 specification.
+
+For complex element types, it computes a complex multiplication, with corner
+cases TBD.
```

Comment:
Turns out that `multiply` in HLO also supports booleans (#262). Let's call this out in the spec (see #264 for an example of how to phrase this).